src/containers/Container.jsx

```diff
@@ -20,6 +21,7 @@ export default function Container() {
                     <Route path="/dashboard" component={Dashboard} />
                     <Route path="/PrivacyPolicy" component={PrivacyPolicy} />
                     <Route path="/about" component={About} />
```

Comment:
Paths should be in a file called `routes.js` and exported like this 

routes.js
```
export const HOME_ROUTE = '/';
export const DASHBOARD_ROUTE = '/dashboard';
export const NOT_FOUND_ROUTE = '/not-found'
```

This way you only import those and avoid silly mistakes like spelling dashboard wrongly on another component or something!